### ISSUE TITLE
Move the image/icon on the x/y position

### ISSUE THREAD

**Issue by Eclextic** [heyjuvi/avizo#71]
It would be nice if we got a way to move the icon displayed on the x and y position, so that the volume icon can be placed next to the slider for example.

**Comment by donovanglover** [comment]
[SwayOSD](https://github.com/ErikReider/SwayOSD) does this by default as well.

**Comment by heyjuvi** [comment]
Agree, this would be cool. For that the image should probably be drawn manually I instead of using `GtkImage`, so it can be freely positioned.

Questions to be answered: How to place the progress bar relative to the image? Should it also be freely movable? Relative to the notifications center?

Feel free to propose something! :)

**Comment by Eclextic** [comment]
I'd propose the following:

The center bar will always be the center no matter what.
It rather is a question of how to place the *image* relative to the slider rather than the *slider* to the image...

Going with image relative to the slider:
- Split the center of the slider's position into 4 diagonals that define the T(op)M(iddle), RM, BM and LM anchorings.
- Anchor the image relative to those

Here is a fitting representation in MS paint cus why not:
![image](https://github.com/user-attachments/assets/048c2f0c-2852-4094-8c07-2c7345dfed0e)

(god this looks horrendous xd)
